types/svelte-range-slider-pips/svelte-range-slider-pips-tests.ts

```diff
@@ -0,0 +1,7 @@
+import RangeSliderPips from 'svelte-range-slider-pips';
```

Comment:
Use the commonjs syntax `import RangeSliderPips = require('svelte-range-slider-pips')`.

Consumers of the package who want to use `import RangeSliderPips from 'svelte-range-slider-pips'` can add `"esModuleInterop": true` (and this is the default with `tsc --init`), but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.